### PR TITLE
Add link to Clock in Lineage OS 18.1

### PR DESCRIPTION
**Pull request by chipotledonut**
## Description

Themed clock icon not working after updating to 2.1.0 in Lineage OS 18.1

:x: Bug fix (non-breaking change which fixes an issue)
:white_check_mark:  Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:x: General change (non-breaking change that doesn't fit the above categories like copyediting)

### Icons linked
* Clock (linked `com.android.deskclock` to `@drawable/clock`)

### CODE REVIEW

File: app/assets/appfilter.xml
Comment (Chefski) [inline app/assets/appfilter.xml:1]: This link is already present. Does the icon not work?

Comment (chipotledonut) [reply app/assets/appfilter.xml:1]: It doesn't work on my phone. I works as an icon pack but if I tap on themed icons home and app drawer it doesn't change,

This is the link already present:

`<item drawableIgnore="true" component="ComponentInfo{com.android.deskclock/com.android.deskclock.DeskClock}" drawable="clock" name="Clock" />`

I guess that _drawableIgnore="true" _ is the problem?

With my commit, I download debug.apk and it's working.

Comment (SuperDragonXD) [reply app/assets/appfilter.xml:1]: Seems to be a weird issue. Other clock apps seem to be applied normally, but not this. I'll try to fix it soon-ish.

Please create a new issue detailing this btw, and maybe check if other clock apps have this issue.

Comment (chipotledonut) [reply app/assets/appfilter.xml:1]: Done. I checked with Simple Clock and same behavior.

**Comment by flame-0** [comment]
https://github.com/LawnchairLauncher/lawnchair/issues/2976

File: app/assets/appfilter.xml
Comment (flame-0) [reply app/assets/appfilter.xml:1]: https://github.com/LawnchairLauncher/lawnchair/issues/2976

**Pull request closed without merging**